src/e2e/specs/dashboard.spec.ts

```diff
@@ -381,3 +381,55 @@ test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Footer`, () =>
     );
   });
 });
+
+test.describe(`${process.env.E2E_TEST_ENV} - Breaches Dashboard - Navigation`, () => {
+  test.beforeEach(async ({ dashboardPage, page }) => {
+    await dashboardPage.open();
+
+    try {
+      await checkAuthState(page);
+    } catch {
+      console.log("[E2E_LOG] - No fxa auth required, proceeding...");
+    }
+  });
+
+  test("Verify that the navigation bar options work correctly", async ({
+    dashboardPage,
+    page,
+  }) => {
+    // link to testrail
+    test.info().annotations.push({
+      type: "testrail",
+      description:
+        "https://testrail.stage.mozaws.net/index.php?/cases/view/2463568",
+    });
+
+    const goToHrefOf = async (aTag: Locator) => {
+      const href = await aTag.getAttribute("href");
+      expect(href).toBeTruthy();
+      await page.goto(href!);
+    };
+
+    //testrail's step 1
+    await dashboardPage.goToDashboard();
+    await goToHrefOf(dashboardPage.settingsPageLink);
+    await expect(page).toHaveURL(/\/settings$/);
+
+    //testrail's step 2
+    await goToHrefOf(dashboardPage.dashboardPageLink);
+    await expect(page).toHaveURL(/.*\/dashboard.*/);
+
+    // testrail's step 3
+    await dashboardPage.goToSettings();
+    const mntrLogo = page.locator("nav a:has(> img)");
+    await goToHrefOf(mntrLogo);
```

Comment:
@mozilloid Does this work with this selector from the page object model?

https://github.com/mozilla/blurts-server/blob/c056d4e497a55734e631945438465c20211add75/src/e2e/pages/dashBoardPage.ts#L89-L92

Still not a huge fan of the xpaths in general, but it's prior art and we can probably clean that up in a future PR by using a better locator. But at least reusing the existing selector in the page object model lets us avoid using ad-hoc ones within the test files here.

Reply:
@pdehaan this gets the image, but not the parent a-tag... Do you want me to add that to the POM?